x-pack/legacy/plugins/apm/cypress/ingest-data/replay.js

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-console */
+
+/**
+ * This script is useful for ingesting previously generated APM data into Elasticsearch via APM Server
+ *
+ * You can either:
+ *  1. Download a static test data file from: https://storage.googleapis.com/apm-ui-e2e-static-data/events.json
+ *  2. Or, generate the test data file yourself by following the steps in: https://github.com/sqren/kibana/blob/master/x-pack/legacy/plugins/apm/cypress/README.md#how-to-generate-static-data
```

Comment:
probably needs to link to elastic/kibana?

Reply:
I'm going to update that to the exact commit hash for the elastic repo when it's merged in